src/main/java/view25d/view/PatchView.java

```diff
@@ -115,20 +120,41 @@ public void updateValueArray(Context context) {
         }
 
         ArrayList<TurtleValue> temp = new ArrayList<TurtleValue>();
+        ArrayList<LinkValue> temp2 = new ArrayList<LinkValue>();
```

Comment:
I know the existing code used `temp` as the variable name, but maybe there is a better/clearer name for this new temporary variable?